web-api/migrations/20220926145400_document.sql

```diff
@@ -12,8 +12,12 @@
 --  You should have received a copy of the GNU Affero General Public License
 --  along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+CREATE TABLE IF NOT EXISTS document (
+    doc_id TEXT NOT NULL PRIMARY KEY
+);
+
 CREATE TABLE IF NOT EXISTS interaction (
-    doc_id TEXT NOT NULL,
+    doc_id TEXT NOT NULL REFERENCES document (doc_id) ON DELETE CASCADE,
```

Comment:
Yes we need new files otherwise the migration will file with the systems we already deployed

Reply:
i asked multiple times, for example in the planning, if we have any database out there which we would break if we do such kind of changes, and the answer was always no. and now you both tell me that this is indeed the case, this is exactly why i asked that question before :( i'll change it accordingly.

Reply:
We didn't had any databases until last week :sweat_smile: